src/test/java/teammates/test/cases/common/SanitizerTest.java

```diff
@@ -44,27 +47,62 @@ public void testSanitizeTextField() {
     
     @Test
     public void testSanitizeForJs() {
-        
+
+        String unsanitized = "\\ \" ' #"; // i.e., [\ " ' #]
+        String expected = "\\\\ \\&quot; \\&#39; \\#"; // i.e., [\\ \&quot; \&#39; \#]
+        String sanitized = Sanitizer.sanitizeForJs(unsanitized);
+
+        assertEquals(null, Sanitizer.sanitizeForJs(null));
+        assertEquals(expected, sanitized);
     }
     
     @Test
     public void testSanitizeForHtml() {
-        
+
+        String unsanitized = "< > \" / ' &"
+                           + "<script>alert('injected');</script>";
+
+        String expected = "&lt; &gt; &quot; &#x2f; &#39; &amp;"
+                        + "&lt;script&gt;alert(&#39;injected&#39;);&lt;&#x2f;script&gt;";
+
+        String sanitized = Sanitizer.sanitizeForHtml(unsanitized);
+        String sanitizedTwice = Sanitizer.sanitizeForHtml(sanitized);
+
+        assertEquals(null, Sanitizer.sanitizeForHtml(null));
```

Comment:
https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true

Also you can optionally adhere to this naming method if it will help to make things more clear as I understand the current naming method might not be as nice sounding

![screen shot 2016-03-22 at 9 01 07 pm](https://cloud.githubusercontent.com/assets/6291947/13975348/514cc0c6-f071-11e5-9db0-c2bbd84fa108.png)